service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/HearingBookingDetailsControllerTest.java

```diff
@@ -70,28 +78,76 @@ void shouldNotReturnAnErrorWhenHearingDateIsSetInDistantFuture() throws Exceptio
         LocalDateTime distantFuture = LocalDateTime.now().plusYears(1000);
 
         AboutToStartOrSubmitCallbackResponse callbackResponse = makeRequest(createHearing(
-            distantFuture, distantFuture.plusDays(1)));
+            distantFuture, distantFuture.plusDays(1)), EMPTY_BOOKING);
+
+        assertThat(callbackResponse.getErrors()).doesNotContain(ERROR_MESSAGE);
+    }
+
+    @Test
+    void shouldOnlyValidateNewBookingsWhenBookingAlreadyExists() throws Exception {
+        LocalDateTime distantFuture = LocalDateTime.now().plusYears(1000);
+        LocalDateTime distantPast = LocalDateTime.now().minusYears(10000);
+
+        AboutToStartOrSubmitCallbackResponse callbackResponse = makeRequest(createHearing(
+            distantFuture, distantFuture.plusDays(1)), createHearing(distantPast, distantPast.plusDays(1)));
 
         assertThat(callbackResponse.getErrors()).doesNotContain(ERROR_MESSAGE);
     }
 
+    @Test
+    void shouldErrorWhenExistingBookingIsUpdatedToPastDate() {
+        LocalDateTime futureDate = LocalDateTime.now().plusDays(5);
+        UUID hearingId = randomUUID();
+
+        CallbackRequest request = callbackRequestWithEditedBooking(futureDate, hearingId);
+
+        AboutToStartOrSubmitCallbackResponse callbackResponse = postMidEvent(request);
+
+        assertThat(callbackResponse.getErrors()).containsOnlyOnce(ERROR_MESSAGE);
+    }
+
+    private CallbackRequest callbackRequestWithEditedBooking(LocalDateTime futureDate, UUID hearingId) {
+        return CallbackRequest.builder()
+                .caseDetails(CaseDetails.builder()
+                    .data(Map.of("hearingDetails", listBookingWithStartDate(hearingId, futureDate.minusDays(10))))
```

Comment:
this is a bit dodgy, what is feature date is defined outside this method in line 99, but this code assumes what is value set there. If line 99 is changed to plusDays(15) then 112 will produce date in future :) This method should encapsulate logic about what is feature and past date